### PR TITLE
38 guest user

### PR DESCRIPTION
**Pull request by meyerhoferc**
* adds test & feature for visitor path
* visitor sees login / create account links when not logged in 
* test verifies navbar links changed based on current_user
* adds some button styling throughout

all tests pass

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -8,8 +8,8 @@
   <%= f.password_field :password %>
 
   <div class="login-submit">
-    <%=f.submit "Login"%>
+    <%=f.submit "Login", class: "waves-effect waves-light btn"%>
```
Comment (CjMoore) [inline app/views/sessions/new.html.erb:11]: Thanks for the styling

File: app/views/carts/show.html.erb [hunk 2]
```diff
@@ -35,3 +35,19 @@
     <% end %>
   </tbody>
 </table>
+
+<div class="user-links">
+  <!--  checkout button goes here -->
+</div>
+
+<% unless current_user %>
```
Comment (CjMoore) [inline app/views/carts/show.html.erb:43]: This is a great way of dealing with not having a checkout button yet.

**Review by CjMoore — APPROVED** [review]
Looks good to me. Ready for second review and merge @valerietrudell

File: app/views/sessions/new.html.erb (hunk 1 above)
Comment (valerietrudell) [inline app/views/sessions/new.html.erb]: Let it be known I'm excited about this button

**Comment by valerietrudell** [comment]
Looks good, I'm going to merge

**Pull request merged**